### PR TITLE
Simplify sub-experiment code

### PR DESCRIPTION
**Pull request by jrapin**
## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the [**CONTRIBUTING**](https://facebookresearch.github.io/nevergrad/contributing.html) document and completed the CLA (see [**CLA**](https://facebookresearch.github.io/nevergrad/contributing.html#contributor-license-agreement-cla)).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: nevergrad/benchmark/experiments.py [hunk 1]
```diff
@@ -94,44 +94,34 @@ def mltuning(seed: tp.Optional[int] = None, overfitter: bool = False, seq: bool
 
 def naivemltuning(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
     """Counterpart of mltuning with overfitting of valid loss, i.e. train/valid/valid instead of train/valid/test."""
-    internal_generator = mltuning(seed, overfitter=True)
-    for xp in internal_generator:
-        yield xp
+    return mltuning(seed, overfitter=True)
```
Comment (jrapin) [inline nevergrad/benchmark/experiments.py:97]: @teytaud no need for more!

Comment (teytaud) [reply nevergrad/benchmark/experiments.py:97]: oh my god so much useless code...

**Review by teytaud — APPROVED** [review]
no idea why the optimizer groups disappeared but ok cool to have them back.

**Comment by jrapin** [comment]
> no idea why the optimizer groups disappeared but ok cool to have them back.

nevermind those, they are created if we run the tests locally, the file is only there as a "FYI"

**Pull request merged**